dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/utils/LogUtils.java

```diff
@@ -183,6 +183,17 @@ public List<String> getAppIdsFromLogFile(@NonNull String logPath) {
         }
     }
 
+    public String getAppIdsFromLogLine(String logLine) {
+        if (StringUtils.isBlank(logLine))
+            return null;
+        Matcher matcher = APPLICATION_REGEX.matcher(logLine);
+        if (matcher.find()) {
+            String appId = matcher.group();
+            return appId;
+        }
+        return null;
+    }
+
```

Comment:
Same, we don't need this. When we kill a DataxTask, we will find applicationId in log file if it is submitted to Yarn.